src/perfcollect/perfcollect

```diff
@@ -1625,6 +1665,15 @@ CTRLC_Handler()
 
 EndCollect()
 {
+    # The user must either use "collect stop" or CTRL+C to stop collection.
+    if [ "$operation" == "stop" ]
+    then 
+        # Recover trace info in the previous program.
+        source $collectInfoFile  # TODO: exit and dispose upon missing file
```

Comment:
It looks like the collection info file lives in the current working directory.  It should live in the temp directory to avoid cases where you can't write to the current working directory.

Reply:
Thought about the same thing but the purpose of collection info file is to locate where the temp dir is. After exiting from the previous program, we totally lost this info.

Reply:
Oh, right - we create a random directory.  You should still be able to find the actual temp directory (/tmp) and put the file in there.

Reply:
You're right I forgot we can use ```/tmp```